### ISSUE TITLE
Questions!

### ISSUE THREAD

**Issue by Youngkl0726** [John-Yao/focal_loss_pytorch#1]
logpt = F.log_softmax(input) # N*H*W,C
UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.

**Comment by John-Yao** [comment]
It is caused by difference of the version of pytorch .
My pytorch version: 0.2.0_4
You can specific the param "dim" according to your version
Moreover, here is a more simply version of FocalLoss:(
https://github.com/ZijunDeng/pytorch-semantic-segmentation/blob/master/utils/misc.py)
